### PR TITLE
Expand cluster testcase

### PR DESCRIPTION
**Pull request by fbalak**
Based on: https://github.com/Tendrl/specifications/pull/254

### CODE REVIEW

File: web/tendrl/cluster_expand.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Test Steps
+==========
+
+.. test_action::
+   :step:
+       From one of already imported hosts run:
+       ``gluster peer probe [new_host]``
+       where ``[new_host]`` is host that is going to be used to expand cluster.
+       Repeat for all new hosts.
```
Comment (ltrilety) [inline web/tendrl/cluster_expand.rst:38]: I would think about to do some work with volumes too. E.g. create a new one on all nodes or expand existing one etc.

Comment (mbukatov) [reply web/tendrl/cluster_expand.rst:38]: True, but that could be covered in another test case.

**Review by mbukatov — COMMENTED** [review]
Looks good, now I need to check if the current description matches currently planed behavior.

**Comment by mbukatov** [comment]
I need to update this to convey a new way of expand.

**Review by mbukatov — APPROVED** [review]
Looks good, thanks for quick update.

**Pull request merged**